installers/win-setup-template.ps1

```diff
@@ -120,13 +124,24 @@ Write-Host "Copy Python binaries to $PythonArchPath"
 Copy-Item -Path ./$PythonExecName -Destination $PythonArchPath | Out-Null
 
 Write-Host "Install Python $Version in $PythonToolcachePath..."
-$ExecParams = Get-ExecParams -IsMSI $IsMSI -PythonArchPath $PythonArchPath
+$ExecParams = Get-ExecParams -IsMSI $IsMSI -IsFreeThreaded $IsFreeThreaded -PythonArchPath $PythonArchPath
 
 cmd.exe /c "cd $PythonArchPath && call $PythonExecName $ExecParams /quiet"
 if ($LASTEXITCODE -ne 0) {
     Throw "Error happened during Python installation"
 }
 
+# print out all files in $PythonArchPath
```

Comment:
Is this debug information intentional?

On my runner it takes like 10s to traverse and print all the files, since it prints all files recursively in this directory. a few thousands lines of logs.

Reply:
File an issue and/or submit a PR. I think I included this to match some existing behavior, not because I thought it was important, but it's been a while and I may be misremembering.

Reply:
https://github.com/actions/setup-python/issues/1097